robot_smach_states/src/robot_smach_states/manipulation/place_designator.py

```diff
@@ -114,10 +117,13 @@ def _resolve(self):
         rospy.logerr("EmptySpotDesignator: Could not find an empty spot")
         return None
 
-    def _is_poi_unoccupied(self, frame_stamped, surface_entity):
+    def _is_poi_unoccupied(self, frame_stamped: FrameStamped, surface_entity: Entity, volume_id: Optional[str] = None):
```

Comment:
this function is starting to become a mess. But I think that is unavoidable in its current format. In the future we should think about moving this to ED and its plugins.